src/components/application-loader/initializers/configLoader.ts

```diff
@@ -17,5 +19,16 @@ export const loadAllConfig: (baseUrl: string) => Promise<void> = async (baseUrl)
   }
   setBackendConfig(backendConfig)
 
+  const currentText = backendConfig.bannerText
+  if (currentText !== '') {
+    const bannerNote = await getNote('banner')
+    const lastAcknowledgedTimestamp = window.localStorage.getItem('bannerTimeStamp') || ''
+    setBanner({
+      text: currentText,
+      timestamp: bannerNote.createtime,
```

Comment:
I don't think, that we should use `createTime`. If you want to change the banner, then you'll have to delete the note and recreate it. Can we use another attribute? `updateTime`? content? And If we already provide the preview text via backend config.. why can't we provide the timestamp. too?

Reply:
Re: Providing the timestamp
We can totally do that, too. But we need the banner redux state anyway to keep track of a dismissed banner in one session. So I tried to minimize the stuff in the backend config. We could simply use the first header of the banner.md file as text and throw it out of backend config entirely, but that would lead to an extra request on every full reload. As opposed to only making that request, when the banner text is set. (That basically makes the text a flag and we could replace it with a simple boolean and get the timestamp and banner text via the request if you prefer.)

Re: updateTime
That is something I was thinking too, currently the note for whatever reason has no updateTime field, but I think we should add that.

Reply:
I don't want the remove the redux state. I want to reduce the number of Ajax calls that are needed to initialize the app

Reply:
It's always a great idea to reduce the amount of startup HTTP requests, therefore I agree that we should include the bannerTimestamp into the config API call.

But in general we should be careful not to totally mix things up by putting everything into an "all-in-one" API-call just to reduce the amount of requests. But as I said above, in this case it's really useful to do so.